src/components/Registration/HelpContactUs.vue

```diff
@@ -51,14 +51,19 @@
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
 import RegistriesContactInfo from '@/components/common/RegistriesContactInfo.vue'
+import FirmContactInfo from '@/components/common/FirmContactInfo.vue'
+import { Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
 
 @Component({
   components: {
-    RegistriesContactInfo
+    RegistriesContactInfo,
+    FirmContactInfo
   }
 })
 export default class HelpContactUs extends Vue {
   helpToggle = false
+  @Getter(useStore) isTypeFirm!: boolean
```

Comment:
It's fine now but we usually have the getters first then follow by local variables. 

For example:
https://github.com/bcgov/business-create-ui/blob/700a8b48d3cc4570d2b060f4be0ddb6c53e273bf/src/views/Registration/RegistrationDefineBusiness.vue#L180

Reply:
Actually, you can remove this line Shaoyun please because you're not using the getter in this file.

Reply:
Fixed, thanks. Resolving.